formbricks/formbricks
# feat: AI Insights: Open Text Questions

## What does this PR do?
- [x] Initialises OpenAI Integration with Formbricks
- [x] Adds AI Insights to the OpenText Responses
- [ ] Finalised the UI and make a reusable component out of it
- [ ] Figure out Caching wrt survey respondants

To Discuss:
- [ ] Markdown will add another library and might not be great with the streaming
- [ ] What paid features (in-app, link, userTargeting) can access AI integration and add the authorization for it

## Type of change

![Screenshot_2023-11-10-17-28-19_1920x1080](https://github.com/formbricks/formbricks/assets/55556994/44f4f522-2253-499b-bf68-22727bd97f3f)


## How should this be tested?
Add your OpenAI Key in env vars, create a survey and populate it with a few open text answers! Now go to the summary page and you'll see a (not so good design currently bec im not good at it haha) button to generate the insights for you based on them.

## Checklist

### Required


### Appreciated
